### ISSUE TITLE
Minecraft Update 1.19.3 with Fabric API 0.68.1 support.

### ISSUE THREAD

**Issue by KotikSol** [Chocohead/OptiFabric#926]
Minecraft updated along with Fabric, and Optifabric is now incompatible with the update.

**Comment by sklynexd** [comment]
Same issue, wanted to play 1.19.3 and can't, there is no version for it yet.

**Comment by y2k04** [comment]
It's not as simple as adding a number to the string. Trust me, I tried. It just crashes on start.

**Comment by itsMarkiS** [comment]
Optifabric is not a full mod, it's an adaptation for optifine to work in fabric
there is no Optifine for 1.19.3 yet, so obviously there wont be optifabric for 1.19.3 as well

**Comment by sklynexd** [comment]
> Optifabric is not a full mod, it's an adaptation for optifine to work in fabric
> there is no Optifine for 1.19.3 yet, so obviously there wont be optifabric for 1.19.3 as well

Yeah but optifine for 1.19.2 works with 1.19.3 if you have forge instead of fabric, so unlucky for fabric users. That’s the reason why I switched to forged.

**Comment by topia** [comment]
> > Optifabric is not a full mod, it's an adaptation for optifine to work in fabric
> > there is no Optifine for 1.19.3 yet, so obviously there wont be optifabric for 1.19.3 as well
> 
> Yeah but optifine for 1.19.2 works with 1.19.3 if you have forge instead of fabric, so unlucky for fabric users. That’s the reason why I switched to forged.

In my understanding, optifine for 1.19.2 doesn't work with 1.19.3. Did you try it yourself?
On optifine discord: https://discord.com/channels/423430686880301056/471762249476734977/1056627106559303810

**Comment by sklynexd** [comment]
> > > Optifabric is not a full mod, it's an adaptation for optifine to work in fabric
> > > there is no Optifine for 1.19.3 yet, so obviously there wont be optifabric for 1.19.3 as well
> > 
> > 
> > Yeah but optifine for 1.19.2 works with 1.19.3 if you have forge instead of fabric, so unlucky for fabric users. That’s the reason why I switched to forged.
> 
> In my understanding, optifine for 1.19.2 doesn't work with 1.19.3. Did you try it yourself? On optifine discord: https://discord.com/channels/423430686880301056/471762249476734977/1056627106559303810

My bad, I thought I had the latest mc version but I have 1.19.2 and newest really doesn’t work.

**Comment by Intrivus** [comment]
> It's not as simple as adding a number to the string. Trust me, I tried. It just crashes on start.

cuz mojang sussy change syntax every version

**Comment by y2k04** [comment]
> > It's not as simple as adding a number to the string. Trust me, I tried. It just crashes on start.
> 
> 
> 
> cuz mojang sussy change syntax every version

fr

**Comment by Intrivus** [comment]
> > > It's not as simple as adding a number to the string. Trust me, I tried. It just crashes on start.
> > 
> > 
> > 
> > cuz mojang sussy change syntax every version
> 
> fr

The basic explanation is like that. It is like coding syntax is changed through every version
